### DISCUSSION TITLE
Installing on Plesk Docker for a PHP user

### DISCUSSION THREAD

**Discussion by Jursdotme** [directus/directus#4155] [Archive]
I find the installation documentation very lacking.

I have a plesk server where i want to install Directus. Is there a step by step guide somewhere? The install instructions in the docs are very vague and assume quite a bit of backend knowledge.

**Comment by benhaynes** [comment]
Is this question about installing the Directus 8 PHP version? If so, this needs to be transferred to: https://github.com/directus/v8-archive

Also, you'll need to be more specific about what is lacking in the docs.
